### ISSUE TITLE
Schema

### ISSUE THREAD

**Issue by carlitux** [apollographql/eslint-plugin-graphql#37]
All documentation on apollo server uses webpack loader and none to generate json file. What is the process o best practice to keep .gql file and the json to get working this lint?

schemaJson field is not well documented or I am missing something?

**Comment by stubailo** [comment]
The examples don't use the linter right now because you need to manually update the schema.json file whenever you change your schema.

However I agree that there is not a good description of how to get the schema. One way is to use the Apollo codegen CLI: https://github.com/apollostack/apollo-codegen

Mind sending a PR that just includes that apollo-codegen command in the README?

**Comment by renatobenks** [comment]
Fine, this plugin it's not working with my eslint.
Follow my lint file bellow:

```javascript
const schemaJson = require('./data/schema.json');

module.exports = {
  parser: 'babel-eslint',
  env: {
    browser: true,
    node: true,
    jest: true
  },
  plugins: [
    'import',
    'flowtype',
    'react',
    'react-native',
    'jsx-a11y',
    'relay',
    'graphql',
  ],
  extends: 'fbjs',
  rules: {
    'comma-dangle': [2, 'always-multiline'],
    quotes: [2, 'single', { allowTemplateLiterals: true }],
    'react/prop-types': 0,
    'react/jsx-no-bind': 0,
    'react/display-name': 0,
    'new-cap': 0,
    'react-native/no-unused-styles': 2,
    'react-native/split-platform-components': 2,
    'react-native/no-inline-styles': 1,
    'react-native/no-color-literals': 0,
    'no-class-assign': 1,
    'no-console': 1,
    'object-curly-spacing': [1, 'always'],
    'flowtype/define-flow-type': 1,
    'flowtype/use-flow-type': 1,
    'import/first': 2,
    'graphql/template-strings': ['error', {
      env: 'relay',
      schemaJson,
    }],
    'graphql/capitalized-type-name': ['error', {
      schemaJson,
    }],
    'graphql/required-fields': ['error', {
      env: 'relay',
      schemaJson,
      requiredFields: ['id']
    }],
  },
  globals: {
    jest: true,
    describe: true,
    it: true,
    test: false,
    expect: true
  }
};
```

I set 3 three rules using graphql plugin, but no one of them it's working to me. I just wanna know what it's happening. This plugin it's not "out of the box" like described on docs?

**Comment by jnwng** [comment]
closing this as apollo-codegen / graphql-cli provide easy ways to get the schema file, and i've added to the README.md to that effect. 

@renatobenks please open another issue if you're still having issues with this plugin.
